INSTALL.md

```diff
@@ -1,8 +1,10 @@
 # Installation Instructions
 
+You can access the latest installer release [here](https://github.com/allen-cell-animated/agave/tags).
```

Comment:
```suggestion
You can access the [latest installer release here](https://github.com/allen-cell-animated/agave/tags).
```

Reply:
Best practice for links is to include the purpose of the link and its destination!